app/code/Magento/QuoteGraphQl/README.md

```diff
@@ -1,4 +1,82 @@
+[Learn more about createEmptyCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/create-empty-cart.html).
+
+- `addSimpleProductsToCart` mutation - allows you to add any number of simple and group products to the cart at the same time.
+  [Learn more about addSimpleProductsToCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/add-simple-products.html).
+
+- `addVirtualProductsToCart` mutation - allows you to add multiple virtual products to the cart at the same time, but you cannot add other product types with this mutation.
+  [Learn more about addVirtualProductsToCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/add-virtual-products.html).
+
+- `applyCouponToCart` mutation - applies a pre-defined coupon code to the specified cart.
+  [Learn more about applyCouponToCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/apply-coupon.html).
+
+- `removeCouponFromCart` mutation - removes a previously-applied coupon from the cart.
+  [Learn more about removeCouponFromCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/remove-coupon.html).
+
+- `updateCartItems` mutation - allows you to modify items in the specified cart.
+  [Learn more about updateCartItems mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/update-cart-items.html).
+
+- `removeItemFromCart` mutation - deletes the entire quantity of a specified item from the cart.
+  [Learn more about removeItemFromCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/remove-item.html).
+
+- `setShippingAddressesOnCart` mutation - sets one or more shipping addresses on a specific cart.
+  [Learn more about setShippingAddressesOnCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/set-shipping-address.html).
+
+- `setBillingAddressOnCart` mutation - sets the billing address for a specific cart.
+  [Learn more about setBillingAddressOnCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/set-billing-address.html).
+
+- `setShippingMethodsOnCart` mutation - sets one or more delivery methods on a cart.
+  [Learn more about setShippingMethodsOnCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/set-shipping-method.html).
+
+- `setPaymentMethodOnCart` mutation - defines which payment method to apply to the cart.
+  [Learn more about setPaymentMethodOnCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/set-payment-method.html).
+
+- `setGuestEmailOnCart` mutation - assigns email to the quest cart.
+  [Learn more about setGuestEmailOnCart mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/set-guest-email.html).
+
+- `setPaymentMethodAndPlaceOrder` mutation - sets the cart payment method and converts the cart into an order. This mutation has been deprecated. Use the `setPaymentMethodOnCart` and `placeOrder` mutations instead.
```

Comment:
Make the fact this has been deprecated stand out.
```suggestion
- `setPaymentMethodAndPlaceOrder` mutation - sets the cart payment method and converts the cart into an order. **This mutation has been deprecated**. Use the `setPaymentMethodOnCart` and `placeOrder` mutations instead.
```

Reply:
Done